### PR TITLE
feat: add docs for license cluster upload

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

File: zh_CN/advanced/http-api.md [hunk 1]
```diff
@@ -2295,3 +2295,31 @@ $ curl -XDELETE --basic -u admin:public 'http://localhost:8081/api/v4/resources/
 {"code":0}
 ```
 
+### License
+
+用于管理EMQX许可证的设施。
```
Comment (thalesmg) [inline zh_CN/advanced/http-api.md]: Warning: machine translation.

Comment (qzhuyan) [reply zh_CN/advanced/http-api.md]: EMQX 软件许可管理

**Review by zmstone — APPROVED** [review]

File: zh_CN/advanced/http-api.md [hunk 2]
```diff
@@ -2295,3 +2295,31 @@ $ curl -XDELETE --basic -u admin:public 'http://localhost:8081/api/v4/resources/
 {"code":0}
 ```
 
+### License
+
+EMQX 软件许可管理。
+
+#### POST /api/v4/license/upload
+
+将一个新的许可证文件上传到集群。 许可证被验证，然后被复制到集群中的所有节点并重新加载。 新的内容被写入节点中配置的相同的文件路径，旧的许可证内容被备份到基本目录中的一个文件，后缀为发生变化时的时间戳。
```
Comment (zmstone) [inline zh_CN/advanced/http-api.md]:
```suggestion
将一个新的许可证文件上传到集群。 许可证被验证，然后被复制到集群中的所有节点并重新加载。 新的内容被写入节点中配置的相同的文件路径，旧的许可证内容被备份到一个文件，该文件后缀为发生变化时的时间戳。
```

File: en_US/advanced/http-api.md [hunk 3]
```diff
@@ -1784,3 +1784,32 @@ $ curl -XDELETE --basic -u admin:public 'http://localhost:8081/api/v4/resources/
...
 {"code":0}
 ```
+
+### License
+
+Facilities for managing the EMQX license.
+
+#### POST /api/v4/license/upload
+
+Uploads a new license file to the cluster.  The license is validated, then it is copied to all nodes in the cluster and reloaded.  The new contents are written to the same file path configured in the node, and the old license contents are backed up to a file in the base directory, suffixed by the timestamp when the change occurred.
```
Comment (zmstone) [inline en_US/advanced/http-api.md]: what's base directory ?

should be data directory instead ?

Comment (thalesmg) [reply en_US/advanced/http-api.md]: I've updated to reflect that it's the same base directory as the already configured license path.

**Review by zmstone — APPROVED** [review]

**Pull request merged**